infrastructure/installer_header.sh

```diff
@@ -3,35 +3,61 @@ echo ""
 echo "Spryker SDK Installer"
 echo ""
 
-# Create destination folder
-DESTINATION=$1
-DESTINATION=${DESTINATION:-/opt/spryker-sdk}
+SELF_UPDATE=0
+DESTINATION="/opt/spryker-sdk"
 
+for i in "$@"; do
+  case "$i" in
+    --self-update|-u) SELF_UPDATE=1; shift 1 ;;
+    *) DESTINATION="${i%/}"
+  esac
+  shift
+done
 
-mkdir -p "${DESTINATION}" &> /dev/null
+if [[ ${SELF_UPDATE} == 0 ]]; then
```

Comment:
In this case and others below`{}` braces are not necessary. Please check and clear them